public/index.html

```diff
@@ -8,6 +8,45 @@
   <title>ROAR!</title>
 </head>
 <body>
-  <h1 class="title">ROAR!</h1>
+  <h1 class="title">Bring attention to problems on the web</h1>
+  <div class="steps">
+    <div class="step">
+      <header>
```

Comment:
The `h3`s should be inside the `header`, no?

Reply:
yes